samples/loadimpactair_script_sample.js

```diff
@@ -0,0 +1,403 @@
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/Media/Images/milehightheader.jpg"},
+		]);
+		http.batch([
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/Media/Images/pattern.png"}
+		]);
+
+		// Collect end time of execution
+		tEnd = (new Date()).getTime();		
+	});
+
+	// Add trend metric data point,
+	// i.e. the time it took the group to execute
+	// and name the data point the same name as the group name
+    tr.add(tEnd - tStart, {tag: "01_landingpage"});
+
+	// Sleep between 5-10 seconds (the amount of time an average user would read before the next step/action)
+	sleep(rnd(5, 10));
+
+	// Second step - go to booking page
+	// Put in group and name group something resaonable
+	// Keep response of reqest in res variable
+	group ("02_toBooking", function () {
+
+		// Collect start time of group execition
+		tStart = (new Date()).getTime();
+
+		res = http.batch([
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/Pages/Booking.aspx"}
+		]);
+
+		http.batch([
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/WebResource.axd?d=pynGkmcFUV13He1Qd6_TZIYhwal-_oKtgDTk-WGCThe4WaERcXrvazAgcpzILVEKBqxXIOn8SqRDbHeXAGVc5w2&t=636161530540000000"},
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/Scripts/WebForms/MsAjax/MicrosoftAjax.js"},
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/Scripts/WebForms/MsAjax/MicrosoftAjaxWebForms.js"},
+		]);
+
+		// Collect end time of execution
+		tEnd = (new Date()).getTime();
+	});
+
+	// Add trend metric data point,
+	// i.e. the time it took the group to execute
+	// and name the data point the same name as the group name
+    tr.add(tEnd - tStart, {tag: "02_toBooking"});
+
+	// Get __VIEWSTATE and __EVENTVALIDATION from the response body
+	var strViewstate = findViewstate(res[0].body);
+    var strEventvalidation = findEventvalidation(res[0].body);
+
+	// Sleep between 5-10 seconds (the amount of time an average user would read before the next step/action)
+	sleep(rnd(5,10));
+
+	// Third step - select departure location
+	// Put in group and name group something resaonable
+	// Use __VIEWSTATE and __EVENTVALIDATION from previous response in this request
+	// The header says x-www-form-urlencoded so we have to urlencode the content of the variables when we use them.
+	// Keep response of reqest in res variable
+	group ("03_departure", function () {
+
+		// Collect start time of group execition
+		tStart = (new Date()).getTime();
+
+		res = http.batch([
+			{"method" : "POST", 
+			 "url" : "http://loadimpactair.guldskeden.se/Pages/Booking", "body" :  "__EVENTTARGET=ctl00%24MainContent%24ddlDeparture&__EVENTARGUMENT=&__LASTFOCUS=&__VIEWSTATE=" + urlencode(strViewstate) + "&__VIEWSTATEGENERATOR=A4196A29&__EVENTVALIDATION=" + urlencode(strEventvalidation) + "&ctl00%24MainContent%24ddlDeparture=1&ctl00%24MainContent%24ddlDestination=-+Select+-", 

```

Comment:
You don't need to manually encode like that, just pass an object and it'll be form encoded.